### DISCUSSION TITLE
add gene name to Human-GEM

### DISCUSSION THREAD

**Discussion by haowang-bioinfo** [SysBioChalmers/Human-GEM#539] [Ideas]
It might be a good idea to include gene names (i.e. gene symbols) into Human-GEM, as a separate field `model.geneNames`.

**Comment by haowang-bioinfo** [comment]
this would introduce redundancy given the existing "geneSymbols" column in `genes.tsv` and they may co-exist a while. But this should help in many things due to the common use of gene symbols in numerous analysis and literature.

**Comment by JonathanRob** [comment]
I think it could be useful to add this field to the default model structure, as you suggest. There may be a potential issue when calling the `annotateGEM` function (part of the release process before generating the SBML file) which I believe might add either a geneSymbols or geneNames field, though I'm not certain.

**Reply by haowang-bioinfo** [reply]
a new field `geneShortNames` is added to Human-GEM in #549, it has now been enabled with compatibility to `importYaml`, `annotateGEM`, RAVEN, Cobra toolbox, and cobrapy

**Comment by haowang-bioinfo** [comment]
added in #549

**Comment by haowang-bioinfo** [comment]
reopen this, because the addition of `geneShortNames` field potentially lead to deprecation of `translateGrRules` function in multiple cases in the repo

**Reply by haowang-bioinfo** [reply]
`translateGrRules ` seems fine
